gprofiler/utils/perf.py

```diff
@@ -19,12 +22,18 @@ def can_i_use_perf_events() -> bool:
         run_process([perf_path(), "record", "-o", "/dev/null", "--", "/bin/true"])
     except CalledProcessError as e:
         # perf's output upon start error (e.g due to permissions denied error)
-        if e.returncode == 255 and (
-            b"Access to performance monitoring and observability operations is limited" in e.stderr
-            or b"perf_event_open(..., PERF_FLAG_FD_CLOEXEC) failed with unexpected error" in e.stderr
+        if not (
+            e.returncode == 255
+            and (
+                b"Access to performance monitoring and observability operations is limited" in e.stderr
```

Comment:
Why not log these errors? Even though we know these as possible errors we'll still be interested in knowing why perf couldn't run and why we're using `itimer` instead of `cpu`

Reply:
This error is legit - gProfiler is used in non-privileged settings as well. We're using `perf` as a generic mean to check for `perf_events` and mmaping pages etc. All "known" errors listed here - I am fine if they trigger, their cause is clear to me and no need to log. Other types of errors - may be other bugs, e.g `perf` failing to start for whatever reason, and those I want logged.